eng/pipelines/common/download-specific-artifact-step.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
```

Comment:
Is this file used somewhere or it was added for testing purposes?

Reply:
No...this is a template that I added so anyone can use it in future. This is very helpful while developing a new pipeline and you want to do quick testing by skipping the build and just downloading the artifacts from previous build.

Reply:
I will change to something generic.